pkg/oci/load_balancer_security_lists.go

```diff
@@ -209,7 +209,7 @@ func getNodeIngressRules(securityList *baremetal.SecurityList, lbSubnets []*bare
 	ingressRules := []baremetal.IngressSecurityRule{}
 
 	for _, rule := range securityList.IngressSecurityRules {
-		if rule.TCPOptions == nil ||
+		if rule.TCPOptions == nil || rule.TCPOptions.SourcePortRange != nil || rule.TCPOptions.DestinationPortRange == nil ||
```

Comment:
The duplication here scares me a bit and would prefer if we could eventually move this to a tested helper function of some kind.